.config/smoldot.dic

```diff
@@ -0,0 +1,144 @@
+143
+ABI
+API
+APIs
+BIP39
+BLAKE2
+Base58
+CLI
+Deserialize/S
+DoS
+ECDH
+Ed25519
+Endian
+Enqueue/S
+FFI
+FNV
+GrandPa
+HMAC
+HashDoS
+IP
+JIT
+JITs
+JSON
+Jaeger
+Kademlia
+Kusama
+LEB128
+LLVM
+LRU
+MERCHANTABILITY
+Merkle
+NPM
+Namespace
+POV
+Polkadot
+PolkadotJS
+Protobuf
+QUIC
+RPC
+SHA-256
+SQLite
+SS58
+Sr25519
+TCP
+TODO
+Trie
+UDP
+UI/S
+UTC
+Unencrypt/D
+VM
+VM's
+VRF
+Wasi
+Wasm
+WebAssembly
+WebSocket/M
+Westend
+Yamux
+aborter
+allocator
+amongst
+benchmarked
+blockchain
+boolean/S
+bootnode
+broadcasted
+bytecode
+checksum
+combinator
+config
+cryptographic
+datas
+deallocate/NXS
+decodable
+decrementation
+decrypt
+dependant
+dereferences
+deserialization
+deserialize/G
+dialer
+durations
+enqueue/D
+enum/SM
+externalities
+extrinsics
+frontend
+functionalities
+hasher
+iff
+includable
+incrementing
+inherents
+instantiation
+intrinsics
+keystore
+lexicographically
+libp2p
+metadata
+multiaddress/S
+multihash
+mutex/SM
+opcode
+orthogonally
+parachain/S
+parathread/S
+pluggable
+poped
```

Comment:
I think this is a real typo and it should be `popped`, no?

Reply:
yep